sources/core.ts

```diff
@@ -717,7 +725,15 @@ export class CommandBuilder<Context> {
                 for (let t = 0; t < arity; ++t)
                     args.push(` #${t}`);
 
-                segments.push(`[${names.join(`,`)}${args.join(``)}]`);
+                const definition = `${names.join(`,`)}${args.join(``)}`;
+
+                if (showOptionList && description) {
+                    optionList.push({definition, description});
+
+                    continue;
```

Comment:
If I understand the code correctly, `commandBuider.usage` still won't show the full usage with the options inlined when `showOptionList` is set to false. (I'd expect it to either be `bin greet [-o, --output #0] <input>` when `showOptionList` is set to `false` and only `bin greet <input>` with all detailed options below when `showOptionList` is set to `true`).

Reply:
It think it will, because it won't `continue` and thus will execute the segment push at line 736. Might benefit from a slight style refactoring though, I'll do that.

Reply:
Oh, right, my bad! Sorry for the false alarm. 🤦